roles/common/templates/etc/pebbles/app_config.j2

```diff
@@ -14,7 +15,51 @@ BASE_URL: 'https://{{ public_ipv4 }}/'
 EXTERNAL_HTTPS_PORT: {{ application_external_https_port }}
 INTERNAL_API_BASE_URL: 'http://api:{{ gunicorn_bind_port_worker }}/api/v1'
 
-{% if enable_shibboleth | bool %}
+{% if enable_shibboleth | d() | bool %}
 ENABLE_SHIBBOLETH_LOGIN: True
 {% endif %}
 PROVISIONING_NUM_WORKERS: {{ provisioning_num_workers }}
+
+{% if sender_email is defined %}
+SENDER_EMAIL = 'sender@example.org'
+{% endif %}
+{% if mail_server is defined %}
+MAIL_SERVER = 'smtp.example.org'
+{% endif %}
+{% if mail_suppress_send | d() | bool %}
+MAIL_SUPPRESS_SEND = True
+{% endif %}
+{% if mail_use_tls | d() | bool %}
+MAIL_USE_TLS = False
+{% endif %}
+
+{% if fake_provisioning | d() | bool %}
+FAKE_PROVISIONING = False
+{% endif %}
+{% if skip_task_queue | d() | bool %}
+SKIP_TASK_QUEUE = False
+{% endif %}
+{% if write_provisioning_logs | d() | bool %}
+WRITE_PROVISIONING_LOGS = True
+{% endif %}
+
+{%  if public_ipv4 is defined %}
```

Comment:
i'm not happy with the public_ipv4, especially as it is not intuitively named (i think it takes a FQDN) but let's leave that for later